CHANGELOG.rst

```diff
@@ -20,6 +20,10 @@ Changes:
   `#33 <https://github.com/hynek/argon2_cffi/pull/33>`_
   `#34 <https://github.com/hynek/argon2_cffi/pull/34>`_
 - To make the change of hash type backward compatible, ``argon2.PasswordHasher.verify()`` now determines the type of the hash and verifies it accordingly.
+- Some of the hash parameters have been made stricter to be closer to the recommendations.
+  The current goal for a hash verification times is around 50ms.
+- To enable hash agility, it's possible to extract hash parameters using ``argon2.extract_parameters()``.
```

Comment:
Maybe. Why do end-users need to be able to extract parameters at all? Because you want to punt on deciding which parameter pair is "needs upgrading" given that time and memory are separate parameters? (I'm not sure why this feature exists because I thought argon2_cffi already exposed a "needs upgrade" facility.)

Reply:
It’s basically just me exposing the plumbing of that facility. And yeah, given that the parameters are kinda independent, it seems at least useful to expose that.

Reply:
OK, cool. Suggested changelog entry: "Allow bespoke decisions about upgrading Argon2 parameters via the ``argon2.extract_parameters()`` function."

Sorry, in hindsight that was really a nit, I'm sure most people wouldn't be confused by "agility" here.